### ISSUE TITLE
IMPORT as

### ISSUE THREAD

**Issue by idelvall** [earthly/earthly#2651]
One community member reported Earthly failing as `Earthfile line 3:0 invalid arguments for IMPORT: [./lib as core]` for this Earthfile:
```
IMPORT ./libs as core

build.environment:
    DO core+INSTALL.BASICS
```
The failure is due to the missing uppercase `AS` but the message is not clear about that, so the user was thinking IMPORT was not working with UDCs 
Adding something like a "Missing AS keyword after IMPORT" would make it more readable

**Comment by idelvall** [comment]
(The same might be happening for other composite keywords we have)

**Comment by nelsam** [comment]
Yeah, I think I'd like to add an error handler that grabs the line output, walks to the line/column the error happened at, and attempts to provide better error output. In this case (if we updated the line/column to point directly at the `as` word):

```go
word := wordAt(line, column)
if upper := strings.ToUpper(word); word != upper && isKeyword(upper) {
    return errors.Wrap(err, "'%v' is unrecognized in this context [hint: did you mean to use the keyword '%v'?]", word, upper)
}
```
